professor.py

```diff
@@ -28,16 +26,35 @@ def main():
     print("#"*80)
     print('\t\t\tWelcome to classroom voter')
     print("#"*80)
+    
+    ip, port = prompt_for_ip()
+    clientSocket = socket.socket()
+    try:
+        clientSocket.connect((ip, port))
+        print('Successful Connection')
+    except socket.error as e:
+        print('Failed Connection: ' + str(e))
 
     poll = prompt_for_poll()
-    ip, port = prompt_for_ip()
     msg = {
         "endpoint": "Announce_poll",
         "Arguments": {
-            "poll": poll.toBytes()
+            "poll": poll.toDict()
         }
     }
-    send_msg_to_ip(ip, port, msg)
+    
+    send_msg(clientSocket, msg)
+    
+    while True:
```

Comment:
Before only one message was ever being sent and then the program stopped execution.  Now we send the questions and then aggregate the responses.